packages/opentelemetry-sdk-trace-base/src/export/SimpleSpanProcessor.ts

```diff
@@ -81,26 +86,17 @@ export class SimpleSpanProcessor implements SpanProcessor {
           globalErrorHandler(error);
         });
 
-    // Avoid scheduling a promise to make the behavior more predictable and easier to test
```

Comment:
Removed this since it seemed to be misleading - a promise is scheduled by `doExport` anyways so there's no way to avoid scheduling one